css/mystyle.css

```diff
@@ -97,12 +96,12 @@ ol, ul {
 }
 
 .description {
-width: 395px;
-padding: 10px;
-margin: 0 auto;
-border: 1px solid rgb(0,0,0);
-background-color: rgb(0,0,0);
-opacity: 0.6;
+	max-width: 395px;
```

Comment:
Same thing here. Changing it to max-width allows it to get smaller when the screen gets smaller.